cypress/e2e/english/page/page.cy.js

```diff
@@ -1,11 +1,38 @@
+const selectors = {
+  postContent: "[data-test-label='post-content']"
+};
+
 describe('Page', () => {
-  before(() => {
-    cy.visit('/thank-you-for-donating');
+  context('General tests', () => {
+    before(() => {
+      cy.visit('/thank-you-for-donating');
+    });
+
+    it('should render', () => {
+      cy.contains(
+        'Please check your email for a donation receipt. Forward it to donors@freecodecamp.org.'
+      );
+    });
   });
 
-  it('should render', () => {
-    cy.contains(
-      'Please check your email for a donation receipt. Forward it to donors@freecodecamp.org.'
-    );
+  context('Embedded videos', () => {
+    before(() => {
+      cy.visit('/embedded-videos-page');
+    });
+
+    it('should render', () => {
+      cy.contains('Embedded Videos Page');
+    });
+
+    it('the final element of the page content should be a `p` element', () => {
+      cy.get(selectors.postContent)
+        .children()
+        .last()
+        .then($el => {
+          const finalEl = $el[0];
+
+          expect(finalEl.tagName.toLowerCase()).to.equal('p');
+        });
```

Comment:
```suggestion
        .should('have.prop', 'tagName', 'P')
```
I think this is a nicer way of expressing the same point. Also, this test is a little mysterious to me. Why is it important that the final element is a paragraph? I think a comment or change to the `it` message would help.

Reply:
This LGTM – thank you for the suggestion.

The reason for this test is that currently, any embedded videos in markdown sections get pushed to the bottom of the content block. It's because the videos aren't modified in place, and are modified before being appended to the parent element, which is the content block for markdown content, rather than a `p` element like it is for content in the rich text editor.

There's a post and page in the Dockerized English Ghost instance that has embedded videos in the rich text editor and in markdown separated by paragraphs. The post and page have the exact same content, and end with a paragraph, so this test is to make sure the current issue where videos get pushed is fixed.

I'll update the `it` statement to make the intention for this test more clear.